controllers/user/controllers/helper/config/rbac.go

```diff
@@ -45,13 +46,40 @@ func GetUsersNamespace(user string) string {
 	return fmt.Sprintf("ns-%s", user)
 }
 
-func GetUserRole() []rbacV1.PolicyRule {
-	return []rbacV1.PolicyRule{
-		{
-			APIGroups: []string{"*"},
-			Resources: []string{"*"},
-			Verbs:     []string{"*"},
-		},
+func GetUserRole(roleType userv1.UserRoleType) []rbacV1.PolicyRule {
+	switch roleType {
+	case userv1.OwnerRoleType:
+		return []rbacV1.PolicyRule{
+			{
+				APIGroups: []string{"*"},
+				Resources: []string{"*"},
+				Verbs:     []string{"*"},
+			},
+		}
+	case userv1.ManagerRoleType:
+		return []rbacV1.PolicyRule{
+			{
+				APIGroups: []string{"*"},
+				Resources: []string{"*"},
+				Verbs:     []string{"*"},
+			},
+		}
+	case userv1.DeveloperRoleType:
+		return []rbacV1.PolicyRule{
+			{
+				APIGroups: []string{"*"},
+				Resources: []string{"*"},
+				Verbs:     []string{"list", "watch", "get"},
+			},
+		}
+	default:
```

Comment:
By default, there should be no permissions.

Reply:
return an empty `[]rbacV1.PolicyRule{}` should be ok.